charts/stable/tt-rss/values.yaml

```diff
@@ -33,13 +36,19 @@ env:
   # @default -- postgresql.postgresqlDatabase value
   TTRSS_DB_NAME: ""
 
+# -- Configure the services for the chart here.
```

Comment:
```suggestion
# -- Configure the services for the chart here.
# @default -- See below.
```
The reason for the `default` is so that we don't have any json dumps shown as `default` in the README.